src/mappingForLepton.ts

```diff
@@ -99,6 +108,17 @@ export function handleTransfer(event: Transfer): void {
   const _nft = loadOrCreateLeptonNFT(event.address, event.params.tokenId);
   _nft.owner = event.params.to;
   _nft.save();
+
+  if (event.params.from.toHex() != ADDRESS_ZERO) {
+    const _leptonSeller = loadOrCreateProfileMetric(event.params.from);
+    _leptonSeller.transferLeptonCount = _leptonSeller.transferLeptonCount.plus(ONE);
+    _leptonSeller.save();
+    
+    const _leptonBuyer = loadOrCreateProfileMetric(event.params.to);
+    _leptonBuyer.transferLeptonCount = _leptonBuyer.transferLeptonCount.plus(ONE);
```

Comment:
This should be a "receiveLeptonCount" param, otherwise, we track the transfer when they send out or receive in to their account..  Might be nice to differentiate